tasks/windows_client.yml

```diff
@@ -0,0 +1,106 @@
+---
+# tasks/windows_client.yml: Deploy various client-side configurations for Sensu-client on Windows
+# This contains all tasks, but these should be moved off into their respective places.
+
+    #- include_vars: "{{ ansible_distribution }}.yml"
+  - include_vars: "{{ ansible_os_family }}.yml"
+
+    # win_package throws an SSL error with a remote source. Using two steps here as a workaround
```

Comment:
Does this happen if you switch `sensu_msi_download_path` to using `https://.....` ? Currently it's just `http://....`

Reply:
seems to work fine... not sure why i had that. perhaps it was an old ansible bug that got fixed since.